src/Journey.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const nunjucks = require('express-nunjucks');
+
```

Comment:
I'm not sure if coupling a specific version of express to the framework is ideal. What if a developer wants to use a specific version of express? However, we don't want to support every version of express. Thoughts?